### DISCUSSION TITLE
SumatraPDF Blurriness Issue on Windows 11 with Dual Screens

### DISCUSSION THREAD

**Discussion by M4rkoKoo** [sumatrapdfreader/sumatrapdf#3687] [Q&A] [answered]
I have two screens on my computer. When SumatraPDF is open on half of the screen, the text becomes blurry and hard to read. This issue gets fixed when I open the window to full screen. Is there any solution to this? The operating system is Windows 11.

**Comment by kjk** [comment]
We don't handle well multiple monitors with different DPIs.

**Reply by hxhue** [reply]
Hi! Is it possible to add a "Restart" button below "File > Exit", so I can restart SumatraPDF when I connect/disconnect my laptop to/from the monitor in my work place? 

I only use one screen at the same time, but still encounter a DPI related problem. My laptop's builtin screen has a different DPI from my monitor. When I use the monitor, the DPI changed but the tabs in SumatraPDF stays in old size. (In my case it's larger than normal, since the builtin screen's scaling is 125% and the monitor's scaling is 100%.) I always close the application and then reopen it. I think the process can be made a bit easier with a restart button.

**Reply by GitHubRulesOK** [reply]
@mouse2-1 
most users would simply close windows and use windows restart when moving about e.g. home to office etc. however the simplest restart is put SumatraPDF shortcut on the task bar then right click "close all windows" followed with left click to reopen

the bonus with shortcut on task bar is it keeps a history of previously opened files for those you use often.

**Comment by M4rkoKoo** [comment]
I changed the scaling to 125% on both screens, which helped a bit. Then I noticed that the blurriness only affects PDF files written with LaTeX... I tried changing the font to Arev 
\usepackage{arev}
 and the problem disappeared.

**Comment by M4rkoKoo** [comment]
I tried viewing the same PDF file on my old laptop, docked in the same docking station, with SumatraPDF displayed on a separate monitor, positioned in the same half-screen manner, and there was no issue. The screenshots show the difference. The old laptop has version 3.3.3, and the new one has 3.4.6. Same settings in Windows: 125% scaling. What else could I try?
![Dell](https://github.com/sumatrapdfreader/sumatrapdf/assets/144607321/19ff73d0-50a2-4e14-a5fe-9f013f643c4f)
![HP](https://github.com/sumatrapdfreader/sumatrapdf/assets/144607321/a26265b9-94da-43e5-ba6c-3ba139d96cfb)

**Reply by M4rkoKoo** [reply]
Actually, the problem occurs in Dell laptop v3.4.6 using only the laptop's builtin screen and resizing the window to a half-screen. Should I downgrade to older version?

**Reply by M4rkoKoo** [reply, marked as answer]
Downgraded to 3.3.3 and looks good. Installed cm-super and looks even better! 
(see:  https://tex.stackexchange.com/questions/1291/why-are-bitmap-fonts-used-automatically )

**Comment by kjk** [comment]
Please try the latest 3.7 [pre-release](https://www.sumatrapdfreader.org/prerelease) — it has many DPI fixes, including for multi-monitor setups with different scaling.

If you still see the issue there, please let me know by opening an [issue](https://github.com/sumatrapdfreader/sumatrapdf/issues).
